### ISSUE TITLE
Bug report: Generated type for binary is incorrect

### ISSUE THREAD

**Issue by lukaskupczyk** [Carminepo2/openapi-ts-rest#94]
Issue: The type for binaries should be `z.custom<File>()` instead of `z.instanceof(File)`

Reproducible Example:

OpenAPI Specification:

```yaml
openapi: "3.0.2"

info:
  title: Example
  version: 1.0.0

paths:
  /test:
    get:
      operationId: test
      responses:
        200:
          description: OK
          content:
            application/json:
              schema:
                type: string
                format: binary
```

Produces:
```typescript
import { initContract } from '@ts-rest/core';
import { z } from 'zod';

const c = initContract();

export const contract = c.router({ test: { method: 'GET', path: '/test', responses: { 200: z.instanceof(File) } } });
```

Instead of:
```typescript
import { initContract } from '@ts-rest/core';
import { z } from 'zod';

const c = initContract();

export const contract = c.router({ test: { method: 'GET', path: '/test', responses: { 200: z.custom<File>() } } });
```

Thanks a lot again for your awesome work!

**Comment by Carminepo2** [comment]
Thank you very much for your contribution! All the reported bug are expected to be fixed in the `1.0.4` version

**Comment by lukaskupczyk** [comment]
Thanks a lot. Everything works fine now. :)
